### PR TITLE
HW2

### PR DESCRIPTION
**Pull request by Soycid**
+ updated readme
+ added repo link

### CODE REVIEW

File: students/jcr2211.md [hunk 1]
```diff
@@ -6,16 +6,17 @@ UNI: jcr2211
...
+
 ## Project Proposal
 
 Chess PGN reader in svelte js (want to make for my own personal blog). May do something more complicated idk unsure. Other ideas include pokemon showdown bot, RLHF for galai, or OSS jackbox clone.
 
-## Project Contributions
-
-Javascript:
-<https://github.com/tomalama/hackbox>
+[SvelteChess](./projects/javascript/svelteChess.md)
```
Comment (timkpaine) [inline students/jcr2211.md:17]: update link path and re-squash 😈

File: students/jcr2211.md [hunk 2]
```diff
@@ -6,16 +6,17 @@ UNI: jcr2211
...
 
+## Project Contributions
 Python:
-<https://github.com/pmariglia/showdown>
-<https://github.com/lowfuel/progrockdiffusion>
-<https://github.com/alembics/disco-diffusion>
+[Pandas](./projects/python/pandas.md)
```
Comment (timkpaine) [inline students/jcr2211.md:21]: update link path and re-squash 😈

**Review by timkpaine — CHANGES_REQUESTED** [review]

File: students/jcr2211.md (hunk 1 above)
Comment (Soycid) [reply students/jcr2211.md]: fixed

File: students/jcr2211.md (hunk 2 above)
Comment (Soycid) [reply students/jcr2211.md]: fixed

**Pull request merged**